src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Navbar from './components/Navbar';
+
+import MainPage from './components/MainPage';
+import LogIn from './components/LogIn';
+
+import Details from './components/Details';
+import Reservations from './components/Reservations';
+import ReserveForm from './components/ReserveForm';
+import AddCars from './components/AddCars';
+import Remove from './components/RemoveCars';
+import Register from './components/signup';
+import ReserveCar from './components/reserveCar';
+import { fetchAllCars } from './store/mainpage_reducer';
+
+function App() {
```

Comment:
- [ ] Throughout your app, you are using the "function" keyword for your functions.  Please, kindly refactor your functions and functional components to ES6 arrow functions.


## Explanation

All over the software industry, the function keyword has been soft deprecated in favor of ES6 arrow functions.

## Tips and extra help

Just do a search for the "function" word in Vscode and you will find all the functions that need to be refactored to ES6.  This should take you about 5-10 minutes, don't worry.

That way, your app will comply with the latest standards. 💪